doc/output-validation.ipynb

```diff
@@ -0,0 +1,830 @@
+    "\n",
+    "### Table of Contents\n",
+    "1. [Imports](#Imports)\n",
+    "2. [Reading the files](#Reading-the-files)\n",
+    "3. [Metrics Validation](#Metrics-Validation )\n",
+    "   - Sumarry Statistics\n",
+    "   - CDF based Metrics\n",
+    "4. [Plots](#Plots)\n",
+    "   - Redshift stacked distribution of the pdfs\n",
+    "   - Zphot vs. Ztrue\n",
+    "   - QQ Plot\n",
+    "   - KS Plot\n",
+    "5. [Summarized Metrics](#Summarized-Metrics)\n",
+    "\n",
+    "-----\n",
+    "\n",
+    "### Observations and Pre requisites\n",
+    "\n",
+    "- Have a rail environment and kernel \n",
+    "\n",
+    "> Rail Install: https://rail-hub.readthedocs.io/en/latest/source/installation.html <br/>\n",
+    "> To install rail was needed to run the folowing command and in the apl cluster `CFLAGS=\"-O2 -std=gnu99\" pip install pytdigest`\n",
+    "> In a short way, it changes the version for the default C used\n",
+    "\n",
+    "- Have a validation set and the true z. You can use pz-server to dowload a validation set and ztrue files with the id `72_pzcompute_results_for_qa_validation`\n",
+    "> Util command to read the output file in the terminal if necessary\n",
+    "> `h5dump -H output1.hdf5`\n",
+    "\n",
+    "- Copy file utils.py from [rail](https://github.com/LSSTDESC/rail/blob/main/examples/evaluation_examples/utils.py)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "514433c9-3f74-4c39-a7c5-d1327b3ca7c1",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "# Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da70e609-6e4a-43e1-b4da-17c7572d6b36",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Pre-requisites\n",
+    "import os\n",
+    "import numpy as np\n",
+    "import tables_io\n",
+    "import qp \n",
+    "import os\n",
+    "##import holoviews as hv\n",
+    "import pandas as pd\n",
+    "import h5py\n",
+    "\n",
+    "from matplotlib import pyplot as plt \n",
+    "from pathlib import Path\n",
+    "#hv.extension('bokeh')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2c52d24-2e5e-4312-a0bd-11571d87974e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Rail modules\n",
+    "import rail\n",
+    "\n",
+    "from rail.core.data import TableHandle, QPHandle, Hdf5Handle, PqHandle\n",
+    "from rail.core.stage import RailStage\n",
+    "from rail.core.util_stages import ColumnMapper, TableConverter\n",
```

Comment:
I had problems with this line. The rail.core.util_stages module is not being found. Anyway, it's not being used on the notebook, so I think it can be removed or commented on.